src/app/views/preferences/preferences.component.html

```diff
@@ -0,0 +1,7 @@
+<app-nav-bar title="Preferences"></app-nav-bar>
+<div class="preferences__list">
+  <a routerLink="/preferences/vehicles" class="preferences__item">Custom Vehicles</a>
+  <a routerLink="/preferences/co2-goals" class="preferences__item">CO<sub>2</sub> Goals</a>
+  <a routerLink="/preferences/manage-data" class="preferences__item">Manage Data</a>
+  <a routerLink="/about" class="preferences__item">About</a>
```

Comment:
Guess those should be bold as well

Reply:
1bf46ad 

I tried it out, but didn't like it. I rather removed more bold font, since it was heavily overused